### ISSUE TITLE
possible pipreqs bug but fixeable from here

### ISSUE THREAD

**Issue by ivanlen** [ivanlen/pipreqsnb#11]
```python
def gaussian_kernel_offset(x1,x2,sig,delta):
    from scipy.spatial.distance import cdist
    k = np.exp((-cdist(x1,x2, 'euclidean')**2/ 2*sig**2))
    offset = np.full((np.shape(x1)[0],np.shape(x2)[0]),delta)
    
    return k + offset
```
will generate the following error.

```bash
ERROR: Failed on file: ./__temp_pipreqsnb_folder/_pipreqsnb_temp_file.py
Traceback (most recent call last):
  File "/home/ivan/coding_projects/env_tf/bin/pipreqs", line 8, in <module>
    sys.exit(main())
  File "/home/ivan/coding_projects/env_tf/lib/python3.7/site-packages/pipreqs/pipreqs.py", line 470, in main
    init(args)
  File "/home/ivan/coding_projects/env_tf/lib/python3.7/site-packages/pipreqs/pipreqs.py", line 409, in init
    follow_links=follow_links)
  File "/home/ivan/coding_projects/env_tf/lib/python3.7/site-packages/pipreqs/pipreqs.py", line 138, in get_all_imports
    raise exc
  File "/home/ivan/coding_projects/env_tf/lib/python3.7/site-packages/pipreqs/pipreqs.py", line 124, in get_all_imports
    tree = ast.parse(contents)
  File "/home/ivan/coding_projects/env_tf/lib/python3.7/ast.py", line 35, in parse
    return compile(source, filename, mode, PyCF_ONLY_AST)
  File "<unknown>", line 111
    from scipy.spatial.distance import cdist
    ^
IndentationError: unexpected indent

```

We can solve this for notebooks removing all indetation from imports

**Comment by ivanlen** [comment]
Check #12, fixed there

**Comment by ivanlen** [comment]
Fixed in #13
